pyne/endl.py

```diff
@@ -0,0 +1,264 @@
+"""Module for parsing and manipulating data from ENDL evaluations.
+
+For the moment, classes and functions in this module only implement the
+specifications relevant to the EEDL (Evaluated Electron Data Library) and the
+EPDL (Evaluated Photon Data Library). The formats are described by the
+following documents:
+
+"ENDL type formats for the Livermore Evaluated Photon Data Library, EPDL"
+https://www.oecd-nea.org/dbdata/data/manual-endf/nds_eval_epdl.pdf
+
+"ENDL type formats for the Livermore Evaluated Electron Data Library, EEDL"
+https://www.oecd-nea.org/dbdata/data/manual-endf/nds_eval_eedl.pdf
+
+For more information, contact Davide Mancusi <davide.mancusi@cea.fr>.
+"""
+from __future__ import print_function, division, unicode_literals
+
+import re
+from warnings import warn
+from collections import namedtuple, defaultdict
+
+import numpy as np
+
+from pyne.utils import QAWarning
+from pyne import rxdata
+import pyne.utils as utils
+from pyne import nucname
+
+warn(__name__ + ' is not yet QA compliant.', QAWarning)
+
+END_OF_TABLE_RE = re.compile(' {71}1')
+
+
+class Library(rxdata.RxLib):
+    """A class for a file which contains multiple ENDL tables."""
+
+    DataTuple = namedtuple('DataTuple', ['yo', 'limits', 'x1', 'data'])
```

Comment:
Probably best to move this to global scope rather than living on the library class.  Also nice use of namedtuple :)